coremltools/converters/libsvm/__init__.py

```diff
@@ -5,9 +5,9 @@
 
 
 from ..._deps import _HAS_LIBSVM
-from . import _libsvm_converter, _libsvm_util
 
 if _HAS_LIBSVM:
+    from . import _libsvm_converter, _libsvm_util
```

Comment:
The CI is failing.
I am not sure is it related to these three fixes.
I would suggest only include the fixes for the `torchvision` ones in this particular PR 😄

Reply:
Yes, let me remove these.

Reply:
Done.